ansible/roles/setup-kafka/tasks/main.yml

```diff
@@ -12,3 +12,11 @@
   when: kafka_id=="1" and item.retention_time is defined  
   tags:
     - processing-kafka
+
+- name: create topics
```

Comment:
Why do we need another block when you can just list in the previous task itself?
```
with_items: 
  - "{{processing_kafka_topics}}"
  - "{{ml_service_topics}}"
```

Reply:
Ok I get it, we have another conditional `ml_kafka_topic_create == true`.